### DISCUSSION TITLE
Is it possible to do multi-class target detection?

### DISCUSSION THREAD

**Discussion by lucianchauvin** [open-edge-platform/anomalib#1975] [Q&A]
Is there an example anywhere, or is it even possible, to do multi-class target detection using anomaly with the openvino inference? Say, I train my model, and having anomaly detection is great but can I now identify which anomaly is being detected if I have multiple targets? Thanks.

**Comment by abc-125** [comment]
As far as I know, anomaly detection models in anomalib don't have such an ability. They calculate just a score, which shows how normal or abnormal the image is.
